test/integration/image-component/default/pages/blurry-placeholder.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Image from 'next/image'
+
+export default function Page() {
+  return (
+    <div>
+      <p>Blurry Placeholder</p>
+      <Image
+        priority
+        id="blurry-placeholder"
+        src="/test.jpg"
+        width="400"
+        height="400"
+        placeholder="blurry"
```

Comment:
Why `blurry` instead of `blur`?

Reply:
Good point, will change to make naming consistent